cpp/src/barretenberg/honk/proof_system/prover_library.cpp

```diff
@@ -412,4 +419,23 @@ template typename honk::flavor::Ultra::Polynomial compute_sorted_list_accumulato
 template void add_plookup_memory_records_to_wire_4<honk::flavor::Ultra>(
     std::shared_ptr<typename honk::flavor::Ultra::ProvingKey>& key, typename honk::flavor::Ultra::FF eta);
 
+template honk::flavor::UltraGrumpkin::Polynomial compute_permutation_grand_product<honk::flavor::UltraGrumpkin>(
```

Comment:
Ugh, maybe this should go in a (stateless) class so we can greatly simplify explicit instantiation? Not asking you to do that, just complaining out loud.